src/main/java/pl/beata/zajecia22_zad1/UserUtils.java

```diff
@@ -0,0 +1,27 @@
+package pl.beata.zajecia22_zad1;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Predicate;
+
+public class UserUtils {
```

Comment:
To wszystko co tu napisałaś powinno być metodami w repozytorium. Wpadłaś w pułapkę klas `Utils`. Generalnie ta odpowiedzialność jest to odpowiedzialność klasy która jest w naszym systemie repozytorium.